examples/esp8266/simplesample_http/simplesample_http.ino

```diff
@@ -0,0 +1,64 @@
+// Please use an Arduino IDE 1.6.8 (once released) or an hourly build from 1/21/2016 or later from
+// https://www.arduino.cc/en/Main/Software
+// Prior builds may experience pre-processor function prototype failures.
+
+#include <ESP8266WiFi.h>
+#include <time.h>
+#include "simplesample_http.h"
+
+char ssid[] = "ssid"; //  your network SSID (name)
+char pass[] = "password";    // your network password (use for WPA, or use as key for WEP)
+int status = WL_IDLE_STATUS;
+
+void setup() {
+	initSerial();
+	initWifi();
+	initTime();   
+}
+
+void loop() {
+	// Run the SimpleSample from the Azure IoT Hub SDK
+	// You must set the connection string in simplesample_http.c
+  	simplesample_http_run();
+}
+
+void initSerial() {
+	// Start serial and initialize stdout
+    Serial.begin(115200);
+	Serial.setDebugOutput(true);
+}
+
+void initWifi() {
+	// Attempt to connect to Wifi network:
+	Serial.print("Attempting to connect to SSID: ");
+	Serial.println(ssid);
+
+	// Connect to WPA/WPA2 network. Change this line if using open or WEP network:
+	status = WiFi.begin(ssid, pass);
+
+	while (WiFi.status() != WL_CONNECTED) {
+	    delay(500);
+	    Serial.print(".");
+	}
+  
+  	Serial.println("Connected to wifi");
+}
+
+void initTime() {
```

Comment:
This one turned out to be a little more complicated.  HTTPAPI is actually too late to set time properly as the SDK uses it quite a bit before the first HTTP call, and there aren't actually other hooks early enough.  I'm also not sure it is a good idea to hide configuration of time from the user by putting it in the library as people may want to configure it themselves, or use time() prior to initializing the library.  I think we will pull time config into the sketch for all boards so it is clear to users time configuration is a prerequisite.